packages/oraidex-common/src/network.ts

```diff
@@ -686,13 +824,39 @@ export const chainInfos: CustomChainInfo[] = [
     },
     bech32Config: defaultBech32Config("noble"),
     features: ["stargate", "ibc-transfer", "no-legacy-stdTx", "ibc-go"],
+    Icon: NobleIcon,
+    IconLight: NobleIcon,
     currencies: [
       {
-        ...NativeUsdcNobleToken,
-        bridgeTo: ["Oraichain"]
+        coinDenom: "USDC",
+        coinMinimalDenom: "uusdc",
+        coinDecimals: 6,
+        coinGeckoId: "usd-coin",
```

Comment:
we should re-use the NativeUsdcNobleToken variable so we dont need to edit in multiple places